src/styles/blocks/header.scss

```
@@ -0,0 +1,74 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    .image {
+      display: block;
+      width: 40px;
+      height: 40px;
+    }
+  }
+
+  .nav {
+    &__list {
+      display: flex;
+      padding: 0;
+
+      list-style: none;
+    }
+
+    &__item {
+      position: relative;
+
+      &:first-child a {
+        margin-left: 0;
+      }
+
+      &:last-child a {
+        margin-right: 0;
+      }
+    }
+
+    &__link {
+      display: block;
+      height: 60px;
+      margin: 0 10px;
+
+      font-family: inherit;
+      font-size: 12px;
+      font-weight: 500;
+      text-decoration: none;
+      text-transform: uppercase;
+      line-height: 60px;
+      color: $text-color;
+
+      transition: $transition;
```

don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

Create a variable for transition duration
For example
```suggestion
    transition: color $transition, transform $transition;
```